modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="alibaba-ai-converting-image-to-qcow2_{context}"]
+= Converting the discovery image to QCOW2 format
+
+Convert the generated ISO to `QCOW2` format before importing it into {alibaba}. 
+
+.Prerequisites
+
+* You have created a cluster and downloaded the discovery image in the {ai-full}.
+
+.Procedure
+
+. Open the command line interface on a Linux machine that is outside the cluster, such as your desktop machine.
++
+[NOTE]
+====
+Ensure that the Linux machine has virtualization flags enabled. For details, see _Additional resources_.
```

Comment:
See here for lead-in link sentence style for non-xref situations: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations